cmd/grpcServer/Dockerfile

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright 2021 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Apply multi-stage build
+FROM golang:alpine AS builder
+WORKDIR /builder
+
+# Copy go source files for build
+COPY go.mod ./
+COPY go.sum ./
+COPY cmd/grpcServer/main.go ./
```

Comment:
Could coalesce three copies into one but I don't believe there's any benefit at the current moment

Reply:
Can you collapse into one? I would like to see how that's done :)

Reply:
Yup! The way I usually have written it before is as `COPY go.mod go.sum cmd/grpcServer/main.go ./`.